packages/long-short-pair/schema.graphql

```diff
@@ -0,0 +1,169 @@
+type LongShortPairContract @entity {
+  "Blockchain address of the contract"
+  id: ID!
+
+  priceIdentifier: String
+
+  "Factory contract used to create this contract"
+  creator: ContractCreator
+
+  deploymentTimestamp: BigInt
+
+  "Address that deployed the contract"
+  deployer: Bytes
+
+  "Address of the contract"
+  address: Bytes!
+
+  "Token used as collateral for this contract"
+  collateralToken: Token
+
+  "Long token for this contract"
+  longToken: Token
+
+  "Short token for this contract"
+  shortToken: Token
+
+  "Expiration timestamp of the LSP contract"
+  expirationTimestamp: BigInt
+
+  "Collateral Per Pair for the LSP contract"
+  collateralPerPair: BigInt
+
+  "Financial product library Library for the LSP contract"
+  financialProductLibrary: Bytes!
+
+  "Custom ancillary data for the LSP contract"
+  customAncillaryData: String
+
+  "Pre-loaded reward for the LSP contract"
+  prepaidProposerReward: BigInt
+
+  "OO bond to be paid by the price proposer"
+  optimisticOracleProposerBond: BigInt
+
+  "OO liveness timer for proposed price"
+  optimisticOracleLivenessTime: BigInt
+
+  "Total collateral within the LSP contract"
+  totalCollateralLocked: BigDecimal
+
+  "Total long tokens current in circulation, issued by the LSP contract"
+  longTokensOutstanding: BigDecimal
+
+  "Total short tokens current in circulation, issued by the LSP contract"
+  shortTokensOutstanding: BigDecimal
+
+  "Contract state of the LSP. Either Open, ExpiredPriceRequested, ExpiredPriceReceived"
```

Comment:
Do you define a constant list of these states anywhere? Might be worth it in `constants`